src/iot/feedTypes.spec.js

```diff
@@ -0,0 +1,58 @@
+import FeedTypes from './feedTypes';
+
+describe('Iot/FeedTypes', function() {
+  let baseRequest;
+  let baseSdk;
+  let expectedHost;
+  let feedTypes;
+  let expectedFeedTypes;
+
+  beforeEach(function() {
+    expectedFeedTypes = fixture.buildList(
+      'feedType',
+      3,
+      {},
+      { fromServer: true }
+    );
+    baseRequest = {
+      get: sinon.stub().resolves(expectedFeedTypes)
+    };
+    baseSdk = {
+      config: {
+        audiences: {
+          iot: fixture.build('audience')
+        }
+      }
+    };
+    expectedHost = faker.internet.url();
+
+    feedTypes = new FeedTypes(baseSdk, baseRequest, expectedHost);
+  });
+
+  afterEach(function() {
+    sinon.restore();
+  });
+
+  describe('getAll', function() {
```

Comment:
Can you refactor these to better match the other tests we have? These are very much unit tests and you're testing multiple units by relying on `toCamelCase`

Here's an example: https://github.com/ndustrialio/contxt-sdk-js/blob/master/src/facilities/costCenters.spec.js#L239

Reply:
I still consider this a unit test, but I'm testing a behavior rather than an implementation. I think we should reserve mocking for external systems like an API, database, or separate module. That would make it an integration test.

For example, this would pass the other tests, leading to a false positive:
```
    return this._request
      .get(`${this._baseUrl}/feeds/types`)
      .then((feedTypes) => {
        const camelCased = toCamelCase(feedTypes);
        return feedTypes; // I accidentally return the wrong variable, but the test passes
      );
```

Reply:
We talked through this on Slack.